skycultures/greek_leidenAratea/description.en.utf8

```diff
@@ -0,0 +1,319 @@
+		<td> &#702;&Omega;&rho;&#943;&omega;&nu; </td>
+		<td> Orion <br> <i>mirrored depiction</i>	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_ori.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Eridanus Eri 		</td>
+		<td>&Pi;&omicron;&tau;&alpha;&mu;&972;&sigmaf; 	</td>
+		<td> the river Eridanus (Latin), an unnamed River (Greek);<br> here a river god is depicted	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_eri.jpg"/> 		</td>
+	</tr>
+	<tr>
+		<td> Lepus Lep 		</td>
+		<td> &Lambda;&alpha;&gamma;&omega;&#771;&sigmaf; </td>
+		<td> Hare <br><i>shown twice: once between Orion's legs, once separately; for the visualisation we have removed the hare between Orion's legs). The separately illustrated hare runs in the direction of writing, but the hare between Orion's legs runs in the direction of the true course of the sun. The constellation figure should therefore be thought of as mirrored in relation to the illustration in the book: this can be seen in the sketch of the hare between Orion's legs.</i>	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_lep.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Canis Maior CMa </td>
+		<td>&Kappa;&#973;&omega;&nu; </td>
+		<td> Dog	<br> mirrored depiction</td>
+		<td> <img width="255" src="descr_pics/aratea_small_cma.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Canis Minor (Procyon) CMi 	</td>
+		<td>&Pi;&rho;&omicron;&kappa;&#973;&omega;&nu; </td>
+		<td> The star in front of the dog <br> mirrored depiction <br>(an asterism of three stars forming a small dog is already mentioned in Eratosthenes, but the name of the figure was always "Prokyon"). Procyon is a Greek term (with "k") latinized.</td>
+		<td> <img width="255" src="descr_pics/aratea_small_cmi.jpg"/> 	</td>
+	</tr>
+	<tr valign="top">
+		<td> Ship Argo Arg </td>
+		<td>&#702;&Alpha;&rho;&gamma;&omega; 	</td>
+		<td> Ship Argo &quot;Argo&quot; <br> mirrored depiction</td>
+		<td> <img width="255" src="descr_pics/aratea_small_arg.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Hydra, Crater, Corvus <br> Hya + Crt + Crv	</td>
+		<td>&#701;&#714;&Upsilon;&delta;&rho;&omicron;&sigmaf;, &Kappa;&rho;&alpha;&tau;&#942;&rho;, &Kappa;&oacute;&rho;&alpha;&xi;  </td>
+		<td> Water Snake, a particular vessel (crater), Raven</td>
+		<td> <img width="255" src="descr_pics/aratea_small_hya.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Centaurus Cen </td>
+		<td>&Kappa;&#941;&nu;&tau;&alpha;&upsilon;&rho;&omicron;&sigmaf; </td>
+		<td> Kentauros 	</td>
+		<td> missing 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Lupus Lup </td>
+		<td> &Theta;&eta;&rho;&#943;&omicron;&nu; </td>
+		<td> monster </td>
+		<td> missing	</td>
+	</tr>
+	<tr valign="top">
+		<td> Ara Ara </td>
+		<td>&Theta;&upsilon;&mu;&iota;&alpha;&tau;&#942;&rho;&iota;&omicron;&nu; </td>
+		<td> Altar 	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_ara.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Corona Australis CrA </td>
```

Comment:
missing comma

Reply:
thank you! 
within writing the list, I once decided to leave the comma out - but you are right, terms like "Leo Leo" look awful; I added all commas (at least, I hope, I did).